bot_drive_node:
  ros__parameters:
    speed:
      x: 0.0
      y: 0.0
      z: 0.0
    rotation:
      x: 0.0
      y: 0.0
      z: 0.0

bot_state_node:
  ros__parameters:
    main_ros: 1
    main: "Idle"
    team: "Blue"
    retry: "None"

flag_grip_node:
  ros__parameters:
    position:
      x: 0.0
      y: 0.0
      z: 0.0
    theta:
      x: 0.0
      y: 0.0
      z: 0.0

flag_grip_model_node:
  ros__parameters:
    max_range: 0.128
    min_range: -0.128

scan_to_scan_filter_chain:
  ros__parameters:
    filter1:
      name: angle
      type: laser_filters/LaserScanAngularBoundsFilter
      params:
        lower_angle: -3.2637657
        upper_angle: 0.0523598776
